src/data/migrations/20170728095755-removeLevelsFieldsFromPools.js

```diff
@@ -0,0 +1,7 @@
+export async function up(r) {
+  return r.table('pools').replace(r.row.without('levels'))
+}
+
+export async function down(r) {
+  return r.table('pools').update({levels: []})
```

Comment:
We don't need this; the data would be unrecoverable once deleted. Would be sufficient (and set clearer expectations) to just include a comment to the effect of "// irreversible migration"